js/gulpfile.js

```diff
@@ -86,9 +86,9 @@ const buildConcurrent = (tasks) => () =>
             .merge(...knownTargets.map((target) =>
                 del(`${targetDir(target, `cls`)}/**`)))));
   
-gulp.task( `test`, gulp.series(getTasks(`test`)));
-gulp.task(`debug`, gulp.series(getTasks(`debug`)));
-gulp.task(`clean`, gulp.parallel(getTasks(`clean`)));
+gulp.task( `test`, gulp.series(/*createTestData,*/ getTasks(`test`)/*, cleanTestData*/));
+gulp.task(`debug`, gulp.series(/*createTestData,*/ getTasks(`debug`)/*, cleanTestData*/));
+gulp.task(`clean`, gulp.parallel(/*cleanTestData,*/ getTasks(`clean`)));
```

Comment:
Should `createTestData` and `cleanTestData` be uncommented so we can remove the arrow files from the repo? I'm thinking these are probably commented now so that other contributors will be able to run the tests without building the Java and C++ impls - if that's the case, maybe we should separate out integration tests, which require the other libraries, and unit tests, which can be run stand-alone?

Reply:
@TheNeuralBit yes, definitely. I put these in to remind us to generate test data on the fly (and how to do it) for the tests, and remove the arrow files from the test directory. Making sure the CI environment had the C++ and Java libs built before the JS tests run was a bit more than I could bite off yesterday on my flight home :)